nixos/configs/vm.nix

```
@@ -36,8 +36,8 @@
   };
 
   nix = {
-    binaryCaches = ["http://127.0.0.1:${toString config.services.spongix.port}"];
-    binaryCachePublicKeys = ["spongix:yNfB2+pMSmrjNyMRWob1oEs4ihPnVKPkECWiDxv1MNI="];
+    binaryCaches = ["http://127.0.0.1:${toString config.services.spongix.port}" "https://cache.iog.io"];
```

```suggestion
    binaryCaches = ["http://127.0.0.1:${toString config.services.spongix.port}"];
```

Our cache is already an upstream of local spongix

---

Before this commit it wasn't in the substituters for spongix, and as I mention in the commit message I don't get cache hits if I don't put it in both

---

Works for me...

```
❯ git diff
nixos/configs/vm.nix:
   nix = {
-    binaryCaches = ["http://127.0.0.1:${toString config.services.spongix.port}" "https://cache.iog.io"];
+    binaryCaches = lib.mkForce ["http://127.0.0.1:${toString config.services.spongix.port}"];

❯ nixos-shell --flake .

[cicero]# nix show-config | grep substituters
substituters = http://127.0.0.1:17745

[cicero]# nix build /nix/store/88j4qzlzhlbzlv3lp176icvxldxmr703-lolcat-100.0.1/
[3/0/19 copied (1.8/63.0 MiB), 0.4/14.2 MiB DL] fetching libunistring-1.0 from http://127.0.0.1:17745
```

---

Not sure what to say, doesn't here :grimacing: